### PR TITLE
Contributions folder

### PR DESCRIPTION
**Pull request by sannecottaar**
Here is one of the things that we discussed during lunch at AGU: having a folder with published and educational contributions (which we don't generally test or are responsible for, but which we can hopefully keep backward compatible for now on). 
For the moment I put the tutorial in there, and the scripts related to our publication. I originally wanted to propose removing the scrips from the other locations, but unfortunately our paper states that the scripts are in the misc directory, so maybe we should keep them there. I'm also fine taking it out of this folder again. 
Cayman, your planetary builder is directly linked to your paper, right? It should probably go in here.

### CODE REVIEW

**Comment by tjhei** [comment]
- are the tutorials now duplicated in /tutorial/ ? 
- is there a reason you didn't move the paper scripts from /misc/ to the new folders?
- do we need to test the new scripts? I think yes!
- if we accept contributions we need to think about licensing issues or clearly state that the contributions folder is not part of BurnMan released under GPL (and exclude it from releases). The current PR is fine of course, because it is our data.

**Comment by sannecottaar** [comment]
> are the tutorials now duplicated in /tutorial/ ?
Yes, but happy to remove them is this setup is approved. And then also change the directories being tested. 

> is there a reason you didn't move the paper scripts from /misc/ to the new folders?
Mainly because they are referenced in the paper as being in misc? I could maybe just put symbolic links in the misc folder?

**Comment by bobmyhill** [comment]
In the readme, I'd change "main authors" to "development team (see Readme.md for a list of developers)". Also, some more guidance for new contributors will probably save time in the long run. For example, each contribution should include a readme.

I agree that contributions should be tested. Perhaps there should be a master script in each contributions directory that runs those *.py files which should be tested, and a test_contributions.sh in the main contributions directory that loops through each contribution, analogous to the test.sh in the main directory.

If previous contributions no longer provide the same result, or if they are broken, they should be removed from the relevant master script, and a note should be added to the readme explaining the reason for the change in result.

Otherwise, this looks good!

**Comment by sannecottaar** [comment]
I cleaned it out a bit more. Moved all the files (this means the scripts won't be in misc anymore, but I left a note). I also moved ipython, which currently won't work for me, so I need to look more into this. I changed test.sh to test our contributions (ignoring ipython stuff)

As for guidance and what we test or not test, I think we should decide this later on experience (if we receive contributions).

**Comment by tjhei** [comment]
Did you add the contributed_scripts/published/CHRU2014/*.pdf files on purpose?

**Comment by sannecottaar** [comment]
Oops, no, let me fix that.

**Comment by tjhei** [comment]
Thanks! Can you please try to squash your commits?

**Comment by sannecottaar** [comment]
I could squash a couple. But I updated changes to ipython and then moved the files, and it is giving me issues with rebasing that. I ran into a sequence of error messages. Sorry, I'll squash more as I go.

**Comment by tjhei** [comment]
I was able to squash all commits without any issues here:
```
git remote update
git checkout contributions_folder
git rebase -i upstream/master   # if "upstream" is the name of the official repo
```
then change all "pick" to "s" except the first. Done.

**Comment by sannecottaar** [comment]
Thanks, I didn't know to use upstream/master, only tried with HEAD~x (which
also showed other people's commits).

**Comment by tjhei** [comment]
So I can not make your ipython files work even with ipython3 installed on my machine. :-( Did you check if they work on cloud.sagemath.com?

**Comment by sannecottaar** [comment]
I tried the old versions, and they work (but then run into the issues with
the new code). I guess something went wrong at some point when trying to
get them compatible with python3 and the testers. I'll make the changes
again to the old files and commit them in this pull request. Sorry for the
delay.

**Comment by tjhei** [comment]
Can you please rename the folder to ``contrib``? That is a standard layout used in many projects and is a little more flexible.

**Comment by tjhei** [comment]
I also don't see the "published" vs "education" as a sensible separation and would dump everything into ``contrib/`` directly.

**Comment by tjhei** [comment]
Finally a question for you: should these files be part of the release .zip file?

**Comment by sannecottaar** [comment]
Better? 
The ipynb are fixed too. 
I think these files could for now be part of the .zip.

**Comment by bobmyhill** [comment]
Looks good to me, if Timo is happy! I think they should be part of the zipped release.

File: misc/readme.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+In case you are looking for any scripts referenced in Cottaar, Heister, Rose and Unterborn (2014), they are now located in:
+contributed_scripts/published/CHRU2014/
```
Comment (tjhei) [inline misc/readme.txt]: this path is not updated

**Comment by tjhei** [comment]
looks good otherwise.

**Comment by tjhei** [comment]
Urgh, I can still not run the stupid ipython notebooks. I think my local installation is messed up. Sigh, python.

**Comment by tjhei** [comment]
Sanne, I got them to work. I have a few changes I want to do, but lets move forward with this. Please change the path (see above) and then we can merge.

**Comment by sannecottaar** [comment]
Done.

**Pull request merged**